content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -22,10 +22,10 @@ $ istioctl analyze --all-namespaces
 
 And that’s it! It’ll give you any recommendations that apply.
 
-For example, if you forgot to enable Istio injection (a very common issue), you would get the following warning:
+For example, if you forgot to enable Istio injection (a very common issue), you would get the following information message:
```

Comment:
nit: How about just say `Info` message? This way user will be able to match with a command line output.
```suggestion
For example, if you forgot to enable Istio injection (a very common issue), you would get the following `Info` message:
```